client/my-sites/domains/domain-management/transfer/transfer-page/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Main from 'calypso/components/main';
+import BodySectionCssClass from 'calypso/layout/body-section-css-class';
+import { getSelectedDomain, isMappedDomain } from 'calypso/lib/domains';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import getPrimaryDomainBySiteId from 'calypso/state/selectors/get-primary-domain-by-site-id';
+import isDomainOnlySite from 'calypso/state/selectors/is-domain-only-site';
+import isPrimaryDomainBySiteId from 'calypso/state/selectors/is-primary-domain-by-site-id';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import { hasLoadedSiteDomains } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+const TransferPage = () => {
+	return (
+		<Main wideLayout>
+			<BodySectionCssClass bodyClass={ [ 'edit__body-white' ] } />
+			The page goes heres
+		</Main>
+	);
+};
+
+const transferPageComponent = connect( ( state, ownProps ) => {
+	const domain = getSelectedDomain( ownProps );
+	const siteId = getSelectedSiteId( state );
+	return {
+		currentRoute: getCurrentRoute( state ),
+		hasSiteDomainsLoaded: hasLoadedSiteDomains( state, siteId ),
+		isAtomic: isSiteAutomatedTransfer( state, siteId ),
+		isDomainOnly: isDomainOnlySite( state, siteId ),
+		isMapping: Boolean( domain ) && isMappedDomain( domain ),
+		isPrimaryDomain: isPrimaryDomainBySiteId( state, siteId, ownProps.selectedDomainName ),
+		primaryDomain: getPrimaryDomainBySiteId( state, siteId ),
+	};
+} )( localize( TransferPage ) );
```

Comment:
I think it's better to prefer the `useI18n` hook from @wordpress/react-i18n over the `localize` HOC.
See here: p1629898294273200-slack-C0BNMNMNG

Reply:
Good call, I'll remove the `localize` HOC so we can use `useI18n` later.